wgpu-hal/src/gles/adapter.rs

```diff
@@ -186,10 +186,20 @@ impl super::Adapter {
         let extensions = gl.supported_extensions();
 
         let (vendor_const, renderer_const) = if extensions.contains("WEBGL_debug_renderer_info") {
+            // context must be current
+            #[cfg(target_os = "emscripten")]
+            if unsafe { super::emscripten::enable_extension("WEBGL_debug_renderer_info\0") } {
+                (GL_UNMASKED_VENDOR_WEBGL, GL_UNMASKED_RENDERER_WEBGL)
+            } else {
+                (glow::VENDOR, glow::RENDERER)
+            }
+            // glow enables WEBGL_debug_renderer_info on w-u-u target.
```

Comment:
W-u-u is great shorthand, but unreadable if you don't know what it means  - maybe just call it the "standard wasm target" or "non-emscripten wasm"